src/main/java/org/sagebionetworks/bridge/udd/config/SpringConfig.java

```diff
@@ -70,9 +70,9 @@ public Table ddbSynapseMapTable() {
         return ddbClient().getTable(environmentConfig().get("synapse.map.table"));
     }
 
-    @Bean(name = "ddbUploadTable")
-    public Table ddbUploadTable() {
-        return ddbClient().getTable(ddbPrefix() + "Upload2");
+    @Bean(name = "ddbSynapseSurveyTable")
+    public Table ddbSynapseSurveyTable() {
+        return ddbClient().getTable(ddbPrefix() + "SynapseSurveyTables");
```

Comment:
Tables plural here in the ddb table name, but Table singular in the code? Can we harmonize?